src/Sylius/Behat/Context/Ui/Admin/ManagingProductVariantsContext.php

```diff
@@ -101,6 +101,15 @@ public function iAddIt()
         $this->createPage->create();
     }
 
+    /**
+     * @When I change its :optionName option to :optionValue
+     */
+    public function iChangeItsOptionTo(string $optionName, string $optionValue): void
+    {
+        $optionName = strtoupper($optionName);
+        $this->updatePage->selectOption($optionName, $optionValue);
```

Comment:
It could be a one-liner
```suggestion
        $this->updatePage->selectOption(strtoupper($optionName), $optionValue);
```